### ISSUE TITLE
ArgumentError:invalid value for Integer(): "08080"

### ISSUE THREAD

**Issue by Stajor** [nbulaj/proxy_fetcher#17]
Ruby version: 2.5.0
Gem version: 0.6.3

Code
```
manager = ProxyFetcher::Manager.new(refresh: false)
manager.cleanup!
manager.refresh_list!(maxtime: '4000')

manager.proxies.each do |proxy|
...
end
```

Error
```
ArgumentError:invalid value for Integer(): "08080"
.../proxy_fetcher-0.6.3/lib/proxy_fetcher/providers/free_proxy_list.rb:28:in `Integer'
.../proxy_fetcher-0.6.3/lib/proxy_fetcher/providers/free_proxy_list.rb:28:in `block in to_proxy'
.../proxy_fetcher-0.6.3/lib/proxy_fetcher/providers/free_proxy_list.rb:26:in `tap'
.../proxy_fetcher-0.6.3/lib/proxy_fetcher/providers/free_proxy_list.rb:26:in `to_proxy'
.../proxy_fetcher-0.6.3/lib/proxy_fetcher/providers/base.rb:10:in `block in fetch_proxies!'
```

**Comment by nbulaj** [comment]
Fixed in d4ebdb440c2bac11f8f90eb085f0a5b7bacf07b0 , thanks!

**Comment by nbulaj** [comment]
Released with 0.6.4
